client/src/Movies/Movie.js

```diff
@@ -5,13 +5,13 @@ export default class Movie extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      movie: null
+      movie: props.match.arguments
```

Comment:
this should be set to null, when we first load the component we set it to null and when component mounts we set this movie